app/src/main/java/com/android/bookswap/utils/ManifestUtils.kt

```diff
@@ -0,0 +1,20 @@
+package com.android.bookswap.utils
+
+import android.content.Context
+import android.content.pm.PackageManager
+
+object ManifestUtils {
+  // Function to retrieve the API key from the manifest file
+  fun getApiKeyFromManifest(context: Context): String? {
```

Comment:
Why use this instead of `BuildConfig.API_KEY` like in ChatGPTApiService or the Map ? 
It's best if every API key is at the same place.